content/articles/qnap-monitoring-environment.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Using SNMP to monitor a QNAP device"
+date: 2020-01-10
+type: "post"
+authors: ["anirudh"]
+tags: ["SNMP","monitoring","grafana","collectd"]
+lede: "It’s very useful to have all our monitoring in a central environment.
+We use Graphite and Grafana to collect metrics from our linux servers, and
+here’s how we added our QNAP file server using SNMP and collectd."
+shortlede: ""
+poster: "poster-qnap-monitoring-environment.jpg"
+thumbnail: "poster-qnap-monitoring-environment.jpg"
+socmediaimg: "socmediaimg-qnap-monitoring-environment.jpg"
+hiliteimg: "poster-qnap-monitoring-environment.jpg"
+poster_sourceurl: "https://www.pinclipart.com/pindetail/Txhmbi_office-management-clipart-performance-monitoring-png-download/"
+poster_source: "Pinclipart.com"
+bookendanimal: "satellite-dish"
+---
+
+At the CTL, we use Graphite and Grafana to monitor all of our server
```

Comment:
I would like to the Graphite and Grafana tools here.

* https://graphite.readthedocs.io/en/latest/
* https://grafana.com/